owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListFragment.kt

```diff
@@ -1099,10 +1123,8 @@ class MainFileListFragment : Fragment(),
             }
 
             R.id.action_remove_file -> {
-                val dialog = RemoveFilesDialogFragment.newInstance(checkedFiles)
-                dialog.show(requireActivity().supportFragmentManager, ConfirmationDialogFragment.FTAG_CONFIRMATION)
-                fileListAdapter.clearSelection()
-                updateActionModeAfterTogglingSelected()
+                filesToRemove = checkedFiles
+                fileOperationsViewModel.showRemoveDialog(filesToRemove)
```

Comment:
I think we don't need `filesToRemove` variable, we can always use `checkedFiles`

Reply:
When the operation is selected from bottom sheet for single file the `checkedFiles` property is not updated. We need the file which was selected to remove when the result for the usecase returns. 

And as stated in the other conversation we cannot directly launch remove dialog for single files because a single file can also be a folder in which case we need the result from usecase.

Reply:
Yes, I wasn't taking that into account hehe, nice! 🥇